app/main/views/applications.py

```diff
@@ -148,6 +148,19 @@ def get_application_by_id(application_id):
     return jsonify(application=application.serializable)
 
 
+@main.route('/applications/<int:application_id>', methods=['POST'])
```

Comment:
should this have /prioritise or something in the URL? it looks like a way to update application content.